### PR TITLE
fix: Add aws cli to system path

### PR DESCRIPTION
**Pull request by sky1122**
*Description of changes:*
Fix cannot directly use aws cli in command line

*Testing done:*
have a windows instance and using windows app test the power shell command

- [X] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/windows-runner-user-data.yaml [hunk 1]
```diff
@@ -59,6 +59,28 @@ tasks:
...
+              if ($currentPath -notlike "*$awsDir*") {
+                  $newPath = "$currentPath;$awsDir"
+                  [Environment]::SetEnvironmentVariable("Path", $newPath, "Machine")
+                  Write-Output "AWS CLI path added to system PATH: $awsDir"
+              } else {
+                  Write-Output "AWS CLI path already in system PATH"
+              }
+          } else {
+              Write-Output "AWS CLI not found"
+          }
```
Comment (pendo324) [inline scripts/windows-runner-user-data.yaml:82]: Do we really need all this logic? If the MSI installation succeeds, we can assume the AWS binary is at the default path.

File: scripts/windows-runner-user-data.yaml [hunk 2]
```diff
@@ -59,6 +59,28 @@ tasks:
           Install-Module -Name AWS.Tools.EC2 -Force
           Install-Module -Name AWS.Tools.AutoScaling -Force
 
+          # Install AWS CLI
+          Invoke-WebRequest -Uri https://awscli.amazonaws.com/AWSCLIV2.msi -OutFile AWSCLIV2.msi
+          Start-Process msiexec.exe -Wait -ArgumentList '/I AWSCLIV2.msi /quiet'
+          aws --version
+          Remove-Item AWSCLIV2.msi
```
Comment (pendo324) [inline scripts/windows-runner-user-data.yaml:66]: Need to merge/rebase from main to remove this from the diff. It's causing a merge conflict.

File: scripts/windows-runner-user-data.yaml (hunk 1 above)
Comment (sky1122) [reply scripts/windows-runner-user-data.yaml:82]: Yes, the default path is `C:\Program Files\Amazon\AWSCLIV2\aws`.

```
    $awsPath = "C:\Program Files\Amazon\AWSCLIV2"
    $currentPath = [Environment]::GetEnvironmentVariable("Path", "Machine")
    $newPath = "$currentPath;$awsPath"
    [Environment]::SetEnvironmentVariable("Path", $newPath, "Machine")
```

Can I change to this? or can we use `C:\Program Files\Amazon\AWSCLIV2\aws` to call the aws cli?

Comment (pendo324) [reply scripts/windows-runner-user-data.yaml:82]: I think that will work, yeah

File: scripts/windows-runner-user-data.yaml (hunk 2 above)
Comment (sky1122) [reply scripts/windows-runner-user-data.yaml]: rebase it

**Review by pendo324 — APPROVED** [review]

**Pull request merged**